### ISSUE TITLE
Create developer map page

### ISSUE THREAD

**Issue by Zimmi48** [rocq-prover/coq.github.io#89]
In the WG, there was discussion about missing visibility of the development team: we said that we should have a page listing current (and past) developers. At some other point, during the discussion on diversity in the Coq community, it was said that it would be good to have a page mapping developers and their topics of interests. Both requirements could probably be adequately combined (and replace the CREDITS file in the main repo). Here is an example of such a page: https://www.sagemath.org/development-map.html

**Comment by gares** [comment]
I like the globe, but I'm afraid the it would look quite empty in our case. I'm not for hiding problems, but does not look like a good advertisement either.

Why not a boring list of faces? https://speckyboy.com/50-webpage-layouts-showcasing-company-teams-employees/

**Comment by ejgallego** [comment]
Indeed an alphabetically ordered list of faces would be the best thing IMO, with a short sentence for each one of us and a link.

**Comment by Zimmi48** [comment]
Faces, sure why not? But something else with show up: lots of males, few females (and that's if we don't include just the current core team), like the "We're cute" page in your showcase.

**Comment by ejgallego** [comment]
Well the map is cute however it would be focused in 3 points so it wouldn't be too nice diversity-wise.

IMO the developer list is more of a communication priority than other facets that of course are also important. But it seems critical to provide such info to the general public.

**Comment by ybertot** [comment]
Making a world map is interesting for communities that have an international spread.  In the case of Coq, the development team is small so this may not be the right target for such a map, but the user community is larger.  If we take authors of maintained developments, it will be more widely spread.  For instance, maintainers of the coq-released opam package have about 50 different mail addresses, mostly in France, but with a few in the US and in Korea.  This may not be the right database to use, though.

**Comment by Zimmi48** [comment]
Indeed, the Sage software has such a spread because it is by itself a distribution of lots of libraries from lots of authors, so it compares more to the Coq package index than to the Coq software itself.

**Comment by Zimmi48** [comment]
Covered by the new Rocq team page.
